docs/index.md

```diff
@@ -0,0 +1,103 @@
+---
+# https://vitepress.dev/reference/default-theme-home-page
+layout: home
+
+hero:
+  name: "Kyma"
+  text: "Power up Kubernetes"
+  tagline: <span class="home-kyma-heavy-bold">Kyma</span> <span class="home-kyma-phonetic">/'ki.ma/</span> is an opinionated set of Kubernetes-based modular building blocks, including all necessary capabilities to develop and run enterprise-grade cloud-native applications. It is the open path to the SAP ecosystem supporting business scenarios end-to-end.
+  image:
+    src: /assets/logo_icon_background.svg
+    alt: Kyma logo
+  actions:
+    - theme: brand
+      text: Why Kyma?
+      link: /README
+    - theme: alt
+      text: Quick Install
+      link: /02-get-started/01-quick-install.html
+features_additonal_description: Who is using <span class="home-kyma-heavy-bold">Kyma</span>?
+features_additonal:
+  - icon: 
+      src: https://raw.githubusercontent.com/kyma-project/website/refs/heads/main/content/adopters/logos/sap.svg
+      width: 180px
+      height: 70px
+    title: SAP
+  - icon: 
+      src: https://raw.githubusercontent.com/kyma-project/website/refs/heads/main/content/adopters/logos/accenture.svg
```

Comment:
All of the icons are taken from the archived repository. In my opinion, it's not a good approach, and please consider moving the icons to this repo, where the code for the VitePress will be.